pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -52,6 +62,15 @@ const (
 	sysPropIotHubConnectionModuleID   = "iothub-connection-module-id"
 	sysPropIotHubEnqueuedTime         = "iothub-enqueuedtime"
 	sysPropMessageID                  = "message-id"
+
+	defaultMessageRetentionInDays = 1
```

Comment:
Can the default be infinity?

Reply:
no it cannot be. See https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-quotas for numbers.